src/pages/Layout/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
```

Comment:
wil je een unit test toevoegen bij nieuwe bestanden. de coverage is nu  maar 50-59% en de coverage moet wel omhoog.

Reply:
Aaaahw yes, helemaal gelijk

Reply:
Ik heb het `Layout` component verwijderd; het design van het register is aangepast en elke pagina moet nu een header hebben.